portfolio/src/main/webapp/index.html

```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+      <meta charset="UTF-8">
+      <title>My Portfolio</title>
+      <link rel="stylesheet" href="style.css">
+      <script src="script.js"></script>
```

Comment:
I think you went from 2 spaces to 4 spaces, but lower down you went from 4 to 2. Personally, I'm used to 2 spaces.

Reply:
Changed it to make it 2 spaces for indentations.